src/hooks/Orderbook/useDrawOrderbook.ts

```diff
@@ -319,6 +319,13 @@ export const useDrawOrderbook = ({
     });
   };
 
+  const [fontFlag, setFontFlag] = useState(0);
```

Comment:
[q] why inc vs bool

Reply:
This callback gets triggered whenever all the currently known fonts finish loading (I think), not necessarily when the font we care about loads, so I just made it an incrementing integer. We could try to be clever and detect our font and bump it then, but I like the simplicity here.

Reply:
in a perfect world this exact message gets added as a comment here. no rush to do it but if you were to put up a super simple follow up PR with that i would stamp it immediately 👀